src/sentry/static/sentry/app/components/tagDistributionMeter/index.jsx

```diff
@@ -154,7 +165,7 @@ const Segment = styled(Link, {shouldForwardProp: isPropValid})`
   border-top-right-radius: ${p => p.last && p.theme.borderRadius};
   border-bottom-right-radius: ${p => p.last && p.theme.borderRadius};
 
-  background-color: ${getColor};
+  background-color: ${p => (p.isOther ? colors[colors.length - 1] : colors[p.index])};
```

Comment:
Is `index` used elsewhere?

Reply:
What do you mean? It's used a couple of times in this component